test_common/harness/imageHelpers.cpp

```diff
@@ -3810,18 +3809,50 @@ bool check_minimum_supported(cl_image_format *formatList,
 			}
 		}
 	}
+}
 
-    for (auto &format: formatsToSupport)
-    {
-        if( !find_format( formatList, numFormats, &format ) )
-        {
-            log_error( "ERROR: Format required by OpenCL %s is not supported: ", version.to_string().c_str() );
-            print_header( &format, true );
-            passed = false;
-        }
-    }
+bool is_image_format_required(cl_image_format format,
+                              cl_mem_flags flags,
+                              cl_mem_object_type image_type,
+                              cl_device_id device)
+{
+	std::vector<cl_image_format> formatsToSupport;
+	build_required_image_formats(flags, image_type, device, formatsToSupport);
+
+	for (auto &formatItr: formatsToSupport)
+	{
+		if (formatItr.image_channel_order == format.image_channel_order &&
+		    formatItr.image_channel_data_type == format.image_channel_data_type)
+		{
+			return true;
+		}
+	}
+
+	return false;
+}
+
+bool check_minimum_supported(cl_image_format *formatList,
```

Comment:
This probably should be moved to `test_conformance/images/clGetInfo/test_loops.cpp` now.

Reply:
As part of this PR? I'm trying to just fix BGRA here.

Reply:
Fair enough. It's become possible as a result of your change but it's not strictly required. That being said, it's not even 2 minutes of copy/paste :). Your call.